demo/__init__.py

```diff
@@ -37,10 +37,12 @@ def execute(actions, name, code_file, make_pipeline_fn, prepare_fn=None, verify_
 def run_demo(name, code_file, make_pipeline_fn, prepare_fn=None, verify_fn=None):
     parser = argparse.ArgumentParser(
         description='What do you want to do with the demo.')
-    parser.add_argument('actions', nargs='*', default=['create'])
+    parser.add_argument('--dbsp_url', required=True)
```

Comment:
can we set a default to localhost?

Reply:
I did this intentionally so that we don't add more demo code that only works with localhost. :) These scripts are invoked from `{create, prepare}_demo_projects.sh`, which sets these arguments using an environment variable.